students/11th/hyojeung/main.html

```diff
@@ -0,0 +1,183 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width">
+    <title>instagram</title>
+    <link href="style/common.css" rel="stylesheet" type="text/css" />
+    <link href="style/main.css" rel="stylesheet" type="text/css" />
+  </head>
+  <body>
+      <!-- 네비게이션 -->
+      <div class="navbox">
+        <nav class="nav">
+          <div class="logobox">
+              <div><img alt="logo" src="img/logoicon_nav.png" width="40"/></div>
```

Comment:
```suggestion
              <div>
                <img alt="logo" src="img/logoicon_nav.png" width="40" />
              </div>
```
부모 자식 관계는 줄바꿈해서 표현해주세요

Reply:
전체적으로 줄바꿈 수정 완료하였습니다!